### ISSUE TITLE
Cannot compile v1.7.2 via bazel

### ISSUE THREAD

**Issue by trivigy** [grpc/grpc#13530]
I am getting a weird error when trying to compile grpc via bazel as a dependency in another project. I am getting the following error. Has anyone seen this before and could suggest a solution?

```bash
ERROR: /home/trivigy/.cache/bazel/_bazel_trivigy/6478f5976e01a17a6f68eacce5ae351f/external/com_google_grpc/BUILD:568:1: C++ compilation of rule '@com_google_grpc//:grpc_base_c' failed (Exit 1)
external/com_google_grpc/src/core/lib/iomgr/fork_posix.c:25:23: fatal error: grpc/fork.h: No such file or directory
```

**Comment by ncteisen** [comment]
Should be fixed by #13574

**Comment by f9rocket** [comment]
I believe this is still broken in v1.8.1.
